### ISSUE TITLE
auto selecting jekyll config file

### ISSUE THREAD

**Issue by GeekinaCave** [jekyll/jekyll-admin#321]
### Description:
Jekyll-admin does not load the configuration file selected for start Jekyll.

### Tell us a bit about yourself:
* Version of JekyllAdmin I'm using : 0.4.0
* Version of Jekyll I'm using : 3.4.1
* Operating System : Windows 10 x64
* Browser : Waterfox 

### Steps to reproduce:
I'm using two Jekyll config files: a _config_dev.yml for local testing and _config.yml for GitHub pages, in that way; i can tinkering with jekyll without mess the live web page.

I use this command for the local testing:

```cmd
jekyll serve --config _config_dev.yml --drafts --incremental
```
then i put in my browser

```html
http://localhost:4000/admin/configuration
```

### I expected the following:
jekyll-admin show the content of the config file selected in jekyll: _config_dev.yml

### But got the following, instead:
jekyll-admin use as configuration reference _config.yml instead the one i mark for run jekyll :confused:

**Comment by benbalter** [comment]
We're using [`Jekyll.sites.first`](https://github.com/jekyll/jekyll-admin/blob/master/lib/jekyll-admin.rb#L44). This *should* be the same config.

**Comment by ashmaroli** [comment]
> We're using Jekyll.sites.first. This should be the same config.

Yes, but nowhere in the API do we set `config = JekyllAdmin.site.config` (I couldn't find it)
But the config file in relation to this issue is defined [here](https://github.com/jekyll/jekyll-admin/blob/master/lib/jekyll-admin/server/configuration.rb#L31-L34)

**Comment by GeekinaCave** [comment]
Guys, i recently update to Jekyll 3.4.3 and Jekyll-admin 0.4.1 and still have the same problem :cry:

**Comment by GeekinaCave** [comment]
Guys, i recently update to Jekyll 3.4.3 and Jekyll-admin 0.4.1 and still have the same problem :cry:

**Comment by ashmaroli** [comment]
Our apologies @GeekinaCave, a proposal to fix this is under review. Please subscribe to #322 to keep track of any developments with it.

**Comment by obalus** [comment]
> [!NOTE]
>
